### PR TITLE
[SRM-943] Updated module to be compatible with drupal 10.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
### JIRA
https://digital-vic.atlassian.net/browse/SRM-943

### Changes
Added D10 compatibility.

### CODE REVIEW

File: baywatch.info.yml [hunk 1]
```diff
@@ -18,9 +17,6 @@ dependencies:
   - password_policy_username:password_policy_username
   - password_strength:password_strength
   - queue_mail:queue_mail
-  - redis:redis
-  - section_purger:section_purger
-  - smtp:smtp
```
Comment (MdNadimHossain) [inline baywatch.info.yml:23]: moved in bay_platform_dependencies

File: composer.json [hunk 2]
```diff
@@ -5,18 +5,18 @@
   "license": "GPL-2.0-or-later",
   "require": {
     "drupal/autologout": "^1.3",
-    "drush/drush": "^10.3",
-    "drupal/coi": "^3",
-    "drupal/purge_queuer_url": "^1.0@RC",
+    "drush/drush": "^11",
+    "drupal/coi": "^4.0@RC",
+    "drupal/purge_queuer_url": "dev-3289191-automated-drupal-10",
```
Comment (MdNadimHossain) [inline composer.json:10]: still no release for D10 compatibility version.

File: src/BaywatchOperation.php [hunk 3]
```diff
@@ -270,22 +270,6 @@ public function exclude_files_path() {
     }
   }
 
-  public function import_default_section_config() {
```
Comment (MdNadimHossain) [inline src/BaywatchOperation.php:273]: moved to bay_platform_dependencies.

**Review by yeniatencio — APPROVED** [review]
LGTM

**Pull request merged**